pixart.js

```diff
@@ -1 +1,23 @@
+var brush = document.querySelector( ".brush" );
 
+// I thought this works for both one and two of the commits?
+// After looking around I could only find a jquery solution...
+var setColorClick = function(){
+    button.addEventListener("click" , function(evt){		    button.addEventListener("click" , function(evt){
```

Comment:
Also, we haven't defined anywhere what button is in your JS. You need to define it first! There are many ways in vanilla js to grab this button element. For example, any of the following would work:
```
var button = document.querySelector("button");
var button = document.querySelector("#set-color")
var button = document.getElementById("set-color");
```